deployment/paiLibrary/paiService/service_management_refresh.py

```diff
@@ -51,7 +51,7 @@ def refresh_all_label(self):
         self.logger.info("Begin to refresh all the nodes' labels")
         machinelist = self.cluster_object_model['layout']['machine-list']
 
-        labels = ['pai-master', 'pai-worker', 'no-drivers', 'no-nodeexporter']
+        labels = ['pai-master', 'pai-worker', 'pai-storage', 'no-drivers', 'no-nodeexporter']
```

Comment:
Should we limit the number of "pai-storage" labelled machines <= 1? If users mislabelled multiple nodes as "pai-storage", there might be a data inconsistence after service restarts.

Reply:
This service can start multiple storage server to support different groups.